package-lock.json

```diff
@@ -803,6 +803,12 @@
       "integrity": "sha512-f5j5b/Gf71L+dbqxIpQ4Z2WlmI/mPJ0fOkGGmFgtb6sAu97EPczzbS3/tJKxmcYDj55OX6ssqwDAWOHIYDRDGA==",
       "dev": true
     },
+    "@types/parse-json": {
```

Comment:
To discuss: what do you think about taking the chance and replace npm with Yarn?

Reply:
I strongly prefer NPM over Yarn. The modern versions of NPM rendered the benefits of Yarn to zero and also you cannot publish to npmjs.com with Yarn so we would need to use NPM anyway (just in the CI, but still two tool).

Reply:
ok :)